packages/builder/src/components/integration/APIEndpointVerbBadge.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  export let verb: string | undefined
+  export let color: string | undefined
+
+  $: displayVerb = verb ? verb.toUpperCase() : undefined
+</script>
+
+{#if displayVerb}
+  <span
+    class="verb-badge"
+    style={color ? `--badge-bg-color:${color}4A;` : undefined}
```

Comment:
NAB: What does the `4A` suffix do?

Reply:
Its for the alpha channel e.g. `#ABABAB4A`. 4A is about 30% opacity